accession/export_csv_utils.py

```diff
@@ -0,0 +1,71 @@
+from accession.models import Accession, Donor, Object
+
+
+def get_object_by_model(model):
+    if model == Accession:
+        return ACC
+
+    elif model == Donor:
+        return DON
+
+    elif model == Object:
+        return OBJ
+
+    else:
+        return NORM
+
+
+class ACC:
+    fields_to_show = []
+    field_serializer_map = {'donor_id': (lambda x: str(Donor.objects.get(id=x)))}
+    field_header_map = {'donor_id': 'donor'}
+
+
+class DON:
+    fields_to_show = [
+        'id', 'salutation', 'first_name', 'middle_name', 'last_name',
+        'organization_name', 'donor_type', 'gender', 'address_1',
+        'address_2', 'city__city', 'state', 'postal_code',
+        'country__country', 'phone_number_1', 'phone_number_2',
+        'email_address', 'comments'
+    ]
+    field_serializer_map = {}
+    field_header_map = {'city__city': 'city', 'country__country': 'country'}
+
+
+class OBJ:
+    fields_to_show = [
+        'id', 'object_number', 'accession_number__accession_number',
+        'object_description', 'related_objects', 'original_numbers',
+        'date_object_creation', 'object_era', 'remarks', 'location_remarks',
+        'construction', 'exhibitions', 'publications', 'provenance',
+        'condition_statement', 'price', 'public_notes',
+        'designer__designer', 'label__label', 'retailer__retailer_name',
+        'retailer_label__retailer_label', 'classification__classification',
+        'country__country', 'gender', 'location__location',
+        'condition__condition', 'material__material',
+        'measurement__measurement', 'type__object_type', 'parts__part',
+        'date_record_added', 'date_record_last_edited'
```

Comment:
I agree it makes it easier to find specific fields. I didn't consider that initially because of how much the length would be extended, but I suppose if I am looking at a very long list of single items per line vs a large list of multiple things per line, the former is less intimidating. I'll make the change.